scripts/docker_build.sh

```diff
@@ -20,5 +21,10 @@
 GIT_ROOT=$(git rev-parse --show-toplevel)
 GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
 
-echo "Setting ORT_VERSION to $GIT_REVISION."
-DOCKER_BUILDKIT=1 docker build -f $GIT_ROOT/Dockerfile -t ort --build-arg ORT_VERSION=$GIT_REVISION $GIT_ROOT
+echo "Setting ORT_VERSION to $GIT_VERSION."
+docker buildx build -f "$GIT_ROOT"/Dockerfile \
+    -t "$ORT_DOCKER" \
+    --build-arg ORT_VERSION="$GIT_VERSION" \
+    --platform linux/amd64 \
+    "$@" \
```

Comment:
This is new. Why is it needed?

Reply:
Build on Arm
Ask @tsteenbe

Reply:
The `--platform linux/amd64`  forces Docker to use linux/amd64 platform by default on Apple Silicon.